KsApi/Service+ServiceType.swift

```diff
@@ -0,0 +1,452 @@
+import Foundation
```

Comment:
I like the idea of making the `Service` class cleaner by creating an extension. I would move the generic functions to the extension instead of the api request functions, though. This would keep the `Service` and `MockService` classes with an identical structure, making the navigation through the code more intuitive.

Reply:
Good point! Done